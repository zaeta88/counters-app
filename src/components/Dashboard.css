.dashboard {
  min-height: 550px;
}