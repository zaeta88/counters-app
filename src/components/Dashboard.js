import React, { Component } from 'react';
import PropTypes from 'prop-types';
import './Dashboard.css';

class Dashboard extends Component {
  constructor(props) {
    super(props);
    this.state = {
    };
  }

  componentDidMount() {
    this.props.handleActiveNav(false)
  }

  render() {
    return (
      <div className="dashboard">
        <div className="site-section pb-0">
          <div className="titlebar">
            <h2 className="title">Dashboard</h2>
          </div>
        </div>
        <div className="wrapper">
          <div className="dashboard-default-message">
            <span>
              <span>Â¡Welcome to the Counters Administrator app!</span>
              <p>Feel free to count as much as you want :)</p>
            </span>
          </div>
        </div>
      </div>
    );
  }
}

Dashboard.propTypes = {
  handleActiveNav:  PropTypes.func
};

export default Dashboard;