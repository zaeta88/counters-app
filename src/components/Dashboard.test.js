/* eslint-disable no-console */
import React from 'react';
import { shallow } from 'enzyme';
import Dashboard from './Dashboard';


describe('React.Component Dashboard', () => {
  it('should render correctly', () => {
    const component = shallow(<Dashboard handleActiveNav = { () => {} } />);
    expect(component).toMatchSnapshot();
  });
});
