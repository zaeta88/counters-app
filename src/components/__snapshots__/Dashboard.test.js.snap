// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React.Component Dashboard should render correctly 1`] = `ShallowWrapper {}`;
